client/dashboard/app/context.tsx

```diff
@@ -56,6 +65,11 @@ const AppContext = createContext< AppConfig >( {
 		me: false,
 		commandPalette: false,
 	},
+	onboardingLinkSourceQueryArg: '',
```

Comment:
All onboarding links use the same "source" param. We could allow it to be customised on a per link basis, but I think that's going to lead to unwanted variation i.e. `?source` params that a different not by design but just because that's the way it was when the onboarding flow got implemented. And then it will get backed in and won't be changable.

So I think it's best to give all ciab onboarding flows the same source query arg from the beginning.